CombatMode/CombatMode.lua

```diff
@@ -362,7 +364,7 @@ function _G.CombatMode_OnUpdate(self, elapsed)
   -- end
 
   -- As the frame watching doesn't need to perform a visibility check every frame, we're adding a stagger
-  if (self.TimeSinceLastUpdate > updateInterval) then
+  if not disabledFreeLookManually and self.TimeSinceLastUpdate > updateInterval then
```

Comment:
Move this to the ShouldCursorBeFreed() function. Put it as another check for the shouldUnlock variable.